packages/editor/src/components/provider/index.js

```diff
@@ -156,15 +156,13 @@ function useBlockEditorProps( post, template, mode ) {
 			return templateBlocks;
 		}
 
-		return postBlocks;
-	}, [
-		templateBlocks,
-		postBlocks,
-		rootLevelPost,
-		post.type,
-		post.id,
-		mode,
-	] );
+		return null;
+	}, [ templateBlocks, rootLevelPost, post.type, post.id, mode ] );
```

Comment:
Just checking, `postBlocks` was kind of redundant anyway given that `post.type` and `post.id` were used to fetch it?